emcc.py

```diff
@@ -1512,7 +1521,10 @@ def repl(m):
         if DEBUG:
           # Copy into temp dir as well, so can be run there too
           shared.safe_copy(memfile, os.path.join(shared.get_emscripten_temp_dir(), os.path.basename(memfile)))
-        return 'memoryInitializer = "%s";' % os.path.basename(memfile)
+        if shared.Settings.CYBERDWARF:
+          return 'memoryInitializer = "%s"; emDebugCDFile = "%s";' % (os.path.basename(memfile), os.path.basename(memfile)[:-3] + "cd")
```

Comment:
This is quite ugly, and I'm not expecting it to ship this way. However I'm not sure of a good way to pass the name of the .cd file